test/controllers/houseConsignment/index/items/GrossWeightControllerSpec.scala

```diff
@@ -88,13 +92,13 @@ class GrossWeightControllerSpec extends SpecBase with AppWithDefaultMockFixtures
       setExistingUserAnswers(emptyUserAnswers)
 
       val request =
-        FakeRequest(POST, GrossWeightAmountRoute)
+        FakeRequest(POST, GrossWeightAmountRouteCheckMode)
           .withFormUrlEncodedBody(("value", validAnswer.toString))
 
       val result = route(app, request).value
 
       status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual onwardRoute.url
+      redirectLocation(result).value mustEqual controllers.routes.HouseConsignmentController.onPageLoad(arrivalId, houseConsignmentIndex).url
```

Comment:
This should use the default onwardRoute.url, you might want to bind the navigator you have built in the AppWithDefaultMock file.

Reply:
Done...